### ISSUE TITLE
Remove Sections from Input File

### ISSUE THREAD

**Issue by timothodge** [bertiniteam/b2#9]
Change parser to ignore "CONFIG", "INPUT", "END;", and allow "CONFIG" and "INPUT" lines to be interspersed.

**Comment by ofloveandhate** [comment]
Nah, we're not going to do this.  Too much work for only a little payoff, esp since we build Python bindings.  We'll keep the current structure.
